orcasecurity/provider.go

```diff
@@ -26,8 +26,8 @@ var (
 	_ provider.Provider = &orcasecurityProvider{}
 )
 
-const apiEndpointEnvName = "ORCASECURITY_API_ENDPOINT"
-const apiTokenEnvName = "ORCASECURITY_API_TOKEN"
+const apiEndpointEnvName = "https://api.orcasecurity.io"
```

Comment:
I'm not sure if these values should be used as standard values:

`https://api.orcasecurity.io` it's orca's API url only in one region - US
Also I don't think we should put any token directly into the repository